lib/customisation.nix

```diff
@@ -106,9 +107,7 @@ rec {
       auto = builtins.intersectAttrs (builtins.functionArgs f) autoArgs;
       finalArgs = auto // args;
       pkgs = f finalArgs;
-      mkAttrOverridable = name: pkg: pkg // {
-        override = newArgs: mkAttrOverridable name (f (finalArgs // newArgs)).${name};
-      };
+      mkAttrOverridable = name: pkg: makeOverridableGeneric (x: x."${name}") f finalArgs;
```

Comment:
It looks like `postProcess` is always applied directly to the output of `f` in `makeOverrideableGeneric`, so why not just pass the composition of `f` and `x: x."${name}"` to `makeOverrideable` here?